spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindReturnRef.java

```diff
@@ -115,153 +123,206 @@ public void visit(Method obj) {
 
     @Override
     public void visit(Code obj) {
-        if (check) {
+        if (check || record) {
             super.visit(obj);
         }
     }
 
     @Override
     public void sawOpcode(int seen) {
 
-        if (!check) {
```

Comment:
Why was this guard clause flipped? It's altered the indentation of the entire (very long) method, which makes it harder to identify the real changes.

Reply:
This guard clause got flipped back, so it's not a problem anymore for the github diff to handle. IMO the code itself was a  bit more logical, but this way it's easy to see the changes.